backend/app/db/database_service.py

```diff
@@ -41,21 +41,27 @@ async def insert_providers_with_links():
     for lp in tqdm(
         local_providers, desc="Inserting provider data and updating country links"
     ):
-        await insert_providers(db_client, data=json.dumps(lp.providers["results"]))
-
-        # Picks the local display priorities
-        local_display_priorities = [
-            {
-                "display_priority": provider["display_priorities"][lp.country_code],
-                "provider_id": provider["provider_id"],
-            }
-            for provider in lp.providers["results"]
-        ]
-        await update_country_providers(
-            db_client,
-            country_code=lp.country_code,
-            providers=json.dumps(local_display_priorities),
-        )
+        if local_providers := lp.providers.get("results"):
+            await insert_providers(db_client, data=json.dumps(local_providers))
+
+            # Picks the local display priorities
+            local_display_priorities = [
+                {
+                    "display_priority": provider["display_priorities"][lp.country_code],
+                    "provider_id": provider["provider_id"],
+                }
+                for provider in lp.providers["results"]
+            ]
+            await update_country_providers(
+                db_client,
+                country_code=lp.country_code,
+                providers=json.dumps(local_display_priorities),
+            )
+        else:
+            print(
+                f"Couldn't find providers in: {lp.country_code} "
+                f"in {insert_providers_with_links.__name__}"
+            )
```

Comment:
Should we keep it consistent with the other log format we have in internal? So we specify what action we did at the end of the message, e.g. "will skip"

Reply:
Yea good idea - also logging should be a thing ø, someone™️ pls